templates/macros/assets.html

```diff
@@ -84,3 +84,11 @@
     {% for s in root.subsections %}{{ self::assets_menu_row(prefix=prefix, section_path=s) }}{% endfor %}
   </ul>
 {% endmacro %}
+{% macro suggestion(id, img, text, url) %}
+<a class="suggestion-item" id="{{ id }}" href="{{ url }}" target="_blank">
+  <div class="suggestion-item__img_wrapper">
+    <img src="{{ img }}" alt="{{ text }} Icon" height="24">
```

Comment:
```suggestion
    <img src="{{ img }}" alt="{{ text }} Icon" width="24" height="24" />
```
You need both width and height for djlint.

Reply:
Thanks! I thought the original message only needed one or the other, though that doesn't make sense now that I think about it... 😅

Fixed in e487e13c5d10087386be2875214ea734c8ef1506.